### PR TITLE
Document parameter value types

### PR DESCRIPTION
**Pull request by nfnt**
**What this PR does / why we need it**:
Documents parameter value types introduced in https://github.com/kudobuilder/kudo/pull/1376

### CODE REVIEW

**Review by kensipe — CHANGES_REQUESTED** [review]
this should be against the `next-release` branch

**Review by porridge — APPROVED** [review]
I noticed this PR literally 1 minute after having merged the `next-release` branch.
Now that the release had happened, I think we should merge straight to `master`. Leaving it up to @kensipe to push the button, LGTM to me.

**Review by meichstedt — APPROVED** [review]

**Pull request merged**
